src/solver.jl

```diff
@@ -130,9 +148,7 @@ Returns a vector of fluxes of the model, if solved.
 flux_vector(flux_balance_analysis(model, ...))
 ```
 """
-flux_vector(model::AbstractMetabolicModel, opt_model)::Maybe{Vector{Float64}} =
-    is_solved(opt_model) ? reaction_variables_matrix(model)' * value.(opt_model[:x]) :
-    nothing
+flux_vector(model::AbstractMetabolicModel, opt_model)::Maybe{Vector{Float64}} = get_solution(:reaction, model, opt_model)
```

Comment:
maybe we could just rename or completely eradicate this now? `flux_` sounds unsystematic to `reaction_`. Flux can be anything

Reply:
will eradicate this in the next PR